modules/store/src/actions_subject.ts

```diff
@@ -13,12 +13,16 @@ export class ActionsSubject extends BehaviorSubject<Action>
   }
 
   next(action: Action): void {
-    if (typeof action === 'undefined') {
+    if (typeof action === 'function') {
+      throw new TypeError(`
+        Dispatch expected an object, instead it received function.
+        If you're using the createAction function, make sure to invoke the function
+        before dispatching the action. E.g. someAction should be someAction().`);
```

Comment:
Nit: Everyone doesn't know what `e.g.` means.

```suggestion
        before dispatching the action. For example, someAction should be someAction().`);
```

Reply:
Done